src/views/Curators/Curator.vue

```diff
@@ -0,0 +1,537 @@
+              let object = {};
+              object.recordNameID = item.name+' ('+item.id+')';
+              object.createdAt = item.createdAt;
+              if (item.creator){
+                object.creator = item.creator.username +' ('+item.creator.id+')';
+              }else{
+                object.creator = "unknown"
+              }
+              this.recordsCreatedCuratorsLastWeek.push(object);
+            });
+          },
+          prepareRecordsInCuration(dataCuration){
+            let userRecords = dataCuration.recordsInCuration;
+            userRecords.forEach(item => {
+              item.fairsharingRecords.forEach(rec => {
+                let object = {};
+                object.curator = item.username;
+                object.recordNameID = rec.name+' ('+rec.id+')';
+                let numMaint = 0;
+                object.recordMaintainers = "none"
+                rec.maintainers.forEach(main => {
+                  if (numMaint > 0){
+                    object.recordMaintainers += ', ' + main.username+' ('+main.id+')';
+                  }else{
+                    object.recordMaintainers = main.username+' ('+main.id+')';
+                  }
+                  numMaint += 1;
+                });
+                this.recordsInCuration.push(object);
+              });
+            });
+          },
+          prepareRecordsWithoutDois(dataCuration){
+            let records = dataCuration.recordsWithoutDois;
+            records.forEach(item => {
+              let object = {};
+              object.recordNameID = item.name+' ('+item.id+')';
+              object.createdAt = item.createdAt;
+              object.updatedAt = item.updatedAt;
+              if (item.creator){
+                object.creator = item.creator.username +' ('+item.creator.id+')';
+              }else{
+                object.creator = "unknown"
+              }
+              if (item.lastEditor){
+                object.lastEditor = item.lastEditor.username +' ('+item.lastEditor.id+')';
+              }else{
+                object.lastEditor = "unknown"
+              }
+              this.recordsWithoutDois.push(object);
+            });
+          },
+          prepareHiddenRecords(dataCuration){
+            let records = dataCuration.hiddenRecords;
+            records.forEach(item => {
+              let object = {};
+              object.recordNameID = item.name+' ('+item.id+')';
+              object.createdAt = item.createdAt;
+              if (item.curator){
+                object.curator = item.curator.username
+              }else{
+                object.curator = 'none'
+              }
+              if (item.creator){
+                object.creator = item.creator.username +' ('+item.creator.id+')';
+              }else{
+                object.creator = "unknown"
+              }
+              this.hiddenRecords.push(object);
+            });
+          }
+      }
+    }
+</script>
+
+<style>
```

Comment:
i'm not 100% sure but this style tagged should be scoped.
```suggestion
<style scoped>
```

Reply:
If I put scoped, the search characters are not changed to white when writing. I didn't put scoped